optimization/loaddata.py

```diff
@@ -0,0 +1,20 @@
+import networkx
```

Comment:
Could you please add `networkx` to [this list]( https://github.com/python-sprints/pyntxos/blob/3767fce7f1b1e37849f34298d399fe71cd9926ec/environment.yml#L7 )?